### PR TITLE
[DO-1824] Delete code

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: business_register/converter/company_converters/ukr_company_full.py [hunk 1]
```diff
@@ -888,7 +888,7 @@ def save_to_db(self, records):
                 self.invalid_data_counter += 1
                 log_records(record, self.LOCAL_FOLDER + 'invalid_companies.txt', self.invalid_data_counter)
                 continue
-            code = name + edrpou
+            code = edrpou + name
```
Comment (roman-tiukh) [inline business_register/converter/company_converters/ukr_company_full.py:891]: do you need this?

File: business_register/converter/company_converters/ukr_company_full.py [hunk 2]
```diff
@@ -980,9 +983,6 @@ def save_to_db(self, records):
             else:
                 self.uptodated_companies.append(company.id)
                 update_fields = []
-                if company.name != name:
-                    company.name = name
-                    update_fields.append('name')
```
Comment (roman-tiukh) [inline business_register/converter/company_converters/ukr_company_full.py:985]: why?

File: business_register/converter/company_converters/ukr_company_full.py [hunk 3]
```diff
@@ -1062,7 +1062,7 @@ def save_to_db(self, records):
         saved_companies = self.bulk_manager.queues['business_register.Company']
         self.bulk_manager.queues['business_register.Company'] = []
         for company in saved_companies:
-            code = company.code
+            code = company.edrpou + company.name
```
Comment (roman-tiukh) [inline business_register/converter/company_converters/ukr_company_full.py:1065]: do you still need this?

File: business_register/converter/company_converters/ukr_company_full.py [hunk 4]
```diff
@@ -1071,7 +1071,7 @@ def save_to_db(self, records):
             self.bulk_manager.commit(Company)
         saved_companies.extend(self.bulk_manager.queues['business_register.Company'])
         for company in saved_companies:
-            code = company.code
+            code = company.edrpou + company.name
```
Comment (roman-tiukh) [inline business_register/converter/company_converters/ukr_company_full.py:1074]: do you still need this?

**Review by roman-tiukh — COMMENTED** [review]
do you delete `code` field from Company model?

File: business_register/converter/company_converters/ukr_company_full.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/company_converters/ukr_company_full.py:891]: yes, it's identificator for assigning additional company tables

File: business_register/converter/company_converters/ukr_company_full.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/converter/company_converters/ukr_company_full.py:985]: we found this company by `edrpou` & `name`. The `name` never will another as `company.name`

File: business_register/converter/company_converters/ukr_company_full.py (hunk 3 above)
Comment (Iva-khar) [reply business_register/converter/company_converters/ukr_company_full.py:1065]: yes, see above

File: business_register/converter/company_converters/ukr_company_full.py (hunk 4 above)
Comment (Iva-khar) [reply business_register/converter/company_converters/ukr_company_full.py:1074]: the same..

File: business_register/converter/company_converters/ukr_company_full.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/company_converters/ukr_company_full.py:891]: yes, it's identificator for assigning additional company tables

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**